packages/app-project/src/helpers/getStaticPageProps/getStaticPageProps.js

```diff
@@ -28,7 +28,7 @@ export default async function getStaticPageProps({ params, query }) {
   const { project } = getSnapshot(store)
   const language = project.primary_language
   const { active_workflows, default_workflow } = project.links
-  
+  const workflowOrder = project.configuration.workflow_order || []
```

Comment:
For consideration add a safety check `project.configuration?.workflow_order`

Reply:
I think configuration will always be defined at this point since it defaults to an object in the store and line 28 grabs a model snapshot, but definitely can do just in case.